src/Mail/Transactional/ProcurationProxyReminderMail.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AppBundle\Mail\Transactional;
+
+use AppBundle\Entity\ProcurationRequest;
+use AppBundle\Utils\PhoneNumberFormatter;
+use AppBundle\Utils\StringCleaner;
+use EnMarche\MailerBundle\Mail\Recipient;
+use EnMarche\MailerBundle\Mail\RecipientInterface;
+use EnMarche\MailerBundle\Mail\Sender;
+use EnMarche\MailerBundle\Mail\SenderInterface;
+use EnMarche\MailerBundle\Mail\TransactionalMail;
+
+final class ProcurationProxyReminderMail extends TransactionalMail
+{
+    public const SUBJECT = 'RAPPEL : votre procuration';
```

Comment:
Not a big deal, but maybe `Votre` instead of `votre` ?

Reply:
the same, it was moved from the old message class